internal/pkg/cli/svc_init.go

```diff
@@ -91,6 +91,11 @@ These messages can be consumed by the Worker Service.`
 "Internet" will configure your service as public.`
 
 	wkldInitImagePrompt = fmt.Sprintf("What's the %s ([registry/]repository[:tag|@digest]) of the image to use?", color.Emphasize("location"))
+
+	fmtStaticSiteInitDirFilePrompt      = "Which " + color.Emphasize("files or directories") + " would you like to upload for %s?\nDirectories are recursively uploaded."
+	staticSiteInitDirFileHelpPrompt     = "Files or (recursive) directories to use for building your static site."
+	fmtStaticSiteInitDirFilePathPrompt  = "What is the custom path to the " + color.Emphasize("directory or file") + " for %s?"
+	staticSiteInitDirFilePathHelpPrompt = "Path to directory or file to use for building your static site."
```

Comment:
is this "notFoundPrompt"? according to `StaticSources(selPrompt, selHelp, notFoundPrompt, notFoundHelp string, pathValidator prompt.ValidatorFunc) ([]string, error)`